controllers/volsync/vshandler.go

```diff
@@ -1253,10 +1253,12 @@ func (v *VSHandler) validateAndProtectSnapshot(
 
 	// Add ownerRef on snapshot pointing to the vrg - if/when the VRG gets cleaned up, then GC can cleanup the snap
 	// Add label to indicate that VolSync should not delete/cleanup this snapshot
```

Comment:
Need to update the comment to explain why the ownered is needed only when the vrg is in the admin namespace.

Reply:
@nirs Commit message added

Reply:
@nirs Commit added